src/components/player/index.js

```diff
@@ -126,6 +85,38 @@ export default class Player extends React.Component {
     }
   }
 
+  handleKeyboardEvents = (e) => {
```

Comment:
Please be consistent - 

```js
handleKeyboardEvents(e) {
  ..
}
```

Reply:
ES6 arrow functions :cry:

Reply:
Changing

Reply:
The repo has an ES6 label though.

Reply:
Yes, the label has ES6 label. But defining class properties as arrow functions isn't the greatest thing I guess. You do not actually get it as a prototype (`Player.prototype.handleKeyboardEvents` wouldn't be defined) which means that if you inherit, you can't call `super.handleKeyboardEvent`. Not to mention the performance drawbacks.

Reply:
:bowing_man: Thanks for explaining, changed. :smile: